src/main/java/racingcar/Cars.java

```diff
@@ -0,0 +1,49 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public void race() {
+        for (int i = 0; i < cars.size(); i++) {
+            Car current = cars.get(i);
+            current.move(RandomGenerator.generate());
+            ResultView.printCarDistanceWithName(current.getDistance(), current.getCarName());
+        }
```

Comment:
foreach를 사용하셔도 좋을 것 같아요! :-) 
```suggestion
         for (Car current : cars) {
            current.move(RandomGenerator.generate());
            ResultView.printCarDistanceWithName(current.getDistance(), current.getCarName());
        }
```

Reply:
넵! 근데 리뷰어님은 for문을 다 forEach로 사용하시는 편인가요? 인덱스를 사용 안하면 더 간결해지긴 하는거 같은데 일반 for문에 비해 뭐가 좋은지 궁금합니다 :)

Reply:
arrayList기준으로 foreach가 for문보다 빠르기는 합니다:-) 
저는 보통 index가 필요해서 구현을 하는경우 외에는 foreach를 사용해요 :-)